pkg/services/auth.go

```diff
@@ -28,15 +27,9 @@ func HasAnyRole(ctx context.Context, roles []model.Role) (bool, error) {
 
 // HasRole authorizes a user as having a given role
 func HasRole(ctx context.Context, role model.Role) (bool, error) {
-	fmt.Println("APP CONTEXT PRINCIPAL", appcontext.Principal(ctx))
-	fmt.Println("APP CONTEXT PRINCIPAL", appcontext.Principal(ctx))
-	fmt.Println("APP CONTEXT PRINCIPAL", appcontext.Principal(ctx))
-	fmt.Println("APP CONTEXT PRINCIPAL", appcontext.Principal(ctx))
-	fmt.Println("APP CONTEXT PRINCIPAL", appcontext.Principal(ctx))
-	fmt.Println("APP CONTEXT PRINCIPAL", appcontext.Principal(ctx))
-	fmt.Println("APP CONTEXT PRINCIPAL", appcontext.Principal(ctx))
-	fmt.Println("APP CONTEXT PRINCIPAL", appcontext.Principal(ctx))
+
 	logger := appcontext.ZLogger(ctx)
+	logger.Info("app context principal", zap.Any("principal", appcontext.Principal(ctx)))
```

Comment:
I would like to standardize our logging format, specifically around usage of caps. It's all over the place. I propose we form a habit that as a simple pass that for short-form logs (less than a sentence) we use all lower case as done in this line.

Reply:
I think I like that, or at least don't start the message with upper case. GO specifically complains about errors starting with upper case.  Could be good to standardize for sure! It's also interesting, the message here is really a little redundant, because it is logging the principal object field in the log as well.